transcom/mymove
# INT B-20437 Send Destination GBLOC to Prime

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20437)

## Summary

This branch updates all Prime API versions to send the destination GBLOC of a move to the Prime, which will allow the prime to get contact info for the destination office. This branch is only for backend API changes, there is currently no space on the frontend for this information to be displayed.

### How to test

1. Access any move in the Prime Simulator role
2. Bring up dev tools and refresh if needed, go to the Network tab.
3.Click on the item with the move's UUID, under the "Order" object, you should now see a "destinationDutyLocationGBLOC" item with a GBLOC string value.


## Screenshots
![primedestgbloc](https://github.com/user-attachments/assets/9c10111e-0b1f-4381-8820-451917dd2b64)
